packages/suite/src/views/guide/OpenGuideFromTooltip.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Icon } from '@trezor/components';
 import { Translation } from '@suite-components';
 import { transparentize } from 'polished';
-import { useGuide } from '@suite-hooks';
+import { useGuideOpenNode } from '@guide-hooks/useGuideOpenNode';
```

Comment:
I would add index.ts into hooks/guide folder to import it like this
```suggestion
import { useGuideOpenNode } from '@guide-hooks';
```

Reply:
https://github.com/trezor/trezor-suite/pull/4426/commits/a8c88dd861508dd477a8cefeb0cf7ea7e2a4e53e